style.css

```diff
@@ -0,0 +1,371 @@
+  }
+
+  .introductionclass {
+    display: flex;
+    flex: 1;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 3em;
+    line-height: 1em;
+    align-items: center;
+    letter-spacing: 0.023125rem;
+    color: #091e42;
+  }
+
+  .second_primary_text {
+    display: flex;
+    flex: 1;
+    width: 74%;
+    height: 5em;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.2em;
+    line-height: 1em;
+    color: #344563;
+  }
+
+  .letconnect {
+    display: flex;
+    flex: 1;
+    width: 7.3rem;
+    height: 1rem;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    line-height: 1rem;
+    color: #7f8cff;
+  }
+
+  .socialmedia {
+    display: flex;
+    flex: 1;
+    width: 10.6rem;
+    height: 1rem;
+  }
+
+  .enabled {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 1rem;
+    gap: 1rem;
+    position: absolute;
+    width: 5.18rem;
+    height: 3rem;
+    left: 6.25rem;
+    background: #fff;
+    border: 0.625rem solid #7f8cff;
+    border-radius: 0.5rem;
+  }
+
+  .Hover {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 1rem;
+    gap: 1rem;
+    position: absolute;
+    width: 5.1rem;
+    height: 3rem;
+    left: 15.75rem;
+    background: #6070ff;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+    border-radius: 0.75rem;
+  }
+
+  .Pressed {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 1rem;
+    gap: 1rem;
+    position: absolute;
+    width: 5.18rem;
+    height: 3rem;
+    left: 25rem;
+    background: #2230d2;
+    border-radius: 1rem;
+  }
+
+  .disabled {
+    width: 3.68rem;
+    height: 2rem;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.2rem;
+    line-height: 2rem;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #5e6c84;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .media {
+    flex: 0 17.017rem;
+    list-style: none;
+    padding: 0;
+    height: 1rem;
+    display: flex;
+  }
+
+  .media li {
+    flex: 1;
+  }
+}
+
+@media (min-width: 768px) {
+  .maincontainer {
+    display: flex;
+    flex-direction: column;
+    width: 92%;
+    height: 82%;
+    padding: 2%;
+    margin: 2%;
+  }
+
+  img {
+    max-width: 92%;
+    max-height: 92%;
+    display: block;
+  }
+
+  .mainmenu {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    align-items: baseline;
+    width: 94%;
+    height: 10%;
+  }
+
+  .LogoPlaceholder {
+    display: flex;
+    flex-direction: row;
+    width: 7rem;
+    height: 10%;
+    padding-left: 5rem;
+    justify-content: start;
+  }
+
+  .itemnavigation {
+    display: flex;
+    flex-direction: row;
+    width: 7rem;
+    height: 10%;
+    justify-content: end;
+  }
+
+  .navitems {
+    padding: 0.3rem;
+  }
+
+  .healdline {
```

Comment:
## Styling ✨✨

- [ ] Kindly, it would be better to replicate the given ```font-colors, font-size, and the spacing``` of your elements as given by the Figma design 😎😎

   Replicating the font `colors, sizes, and alignments` from the Figma design will help ensure that the final product looks consistent and polished.  😎😎

### You can check the comparison below 👇 👇 
![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/102757126/a7f51e50-7013-462e-a2dc-108f6be2cc28)

vs 
![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/102757126/710c4052-1d86-4027-8baf-b604baa68572)